app/models/thredded/user_topic_follow.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+module Thredded
+  class UserTopicFollow < ActiveRecord::Base
+    REASON_MANUAL = 'manual'
```

Comment:
Well, it's the standard way of doing enums in Rails. Out of the box it gives you validations, scopes, and a method that enumerates over all the values.